app/src/appNavigator.tsx

```diff
@@ -0,0 +1,18 @@
+import { createStackNavigator, createAppContainer } from 'react-navigation';
+import { ScheduleScreen } from './screens/scheduleScreen/scheduleScreen';
+import { SpeakerScreen } from './screens/speakers/speakerScreen';
+
+const AppNavigator = createStackNavigator(
+  {
+    Home: ScheduleScreen,
+    Speaker: SpeakerScreen,
+  },
+  {
+    initialRouteName: 'Home',
+    headerMode: 'none',
+  },
+);
+
+const AppContainer = createAppContainer(AppNavigator);
+
+export default AppContainer;
```

Comment:
I think we should convert this to a named export

Reply:
Done :D

Reply:
Thanks :)